perception/hashers/video/scenes.py

```diff
@@ -70,7 +70,13 @@ def __init__(
         self.similarity_threshold = similarity_threshold
 
     def compute_batches(
-        self, filepath, errors="raise", hash_format="base64", batch_size=10
+        self,
+        filepath,
+        errors="raise",
+        hash_format="base64",
+        batch_size=10,
+        keep_frame=False,
+        # TODO: missing **kwargs support for read_video...if this class is used for anything in practice, benchmarking?..?
```

Comment:
so this `compute_batches` doesn't match the signature of the hashingtoolkit video one fwiw